content/updates/2020-09/2-New-user-interface-for-Quick-Exports.md

```diff
@@ -0,0 +1,22 @@
+---
+pim_announcement_img: ./img/Quick-export-new-UI_Export-with-labels.png
+pim_announcement_alt_img: "A new user interface for Quick Exports"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# A new user interface for Quick Exports
+::: meta-data type="New" features="Collaboration, Productivity" available="mid-September" link-to-doc="../articles/quick-export.html"
+:::
+
+For a better user experience, we created a new user interface to make it easier for you to generate a quick export from the product grid. This improvement follows the addition of a new option for all product exports, a new button called  "With labels" (see announcement [here](2020-09.html#display-labels-in-your-product-exports)).  
+![New Export Profile option](../img/Quick-export-new-UI_Export-with-labels.png)
+
+To save you time, the custom choices you make for your XLSX or CSV quick export are saved. So next time you click on `Quick Export`, the last export configuration will be displayed.  
+
+We hope you will enjoy it :wink:
+
+::: more
+[How to quick export your products?](../articles/quick-export.html)   
```

Comment:
For this article, which language is used to export data? the locale of the user? the locale selected in the grid?

Reply:
Yes, the locale selected in the grid.